lib/services/cloud-codesign-service.ts

```diff
@@ -89,7 +89,12 @@ export class CloudCodesignService extends CloudService implements ICloudCodesign
 
 		if (!codesignResult.buildItems || !codesignResult.buildItems.length) {
 			// Something failed.
-			const err: any = new Error(`Codesign failed. Reason is: ${codesignResult.errors}.`);
+			let errText = codesignResult.errors;
```

Comment:
what's the type of `errors`, if it can be `null` or `undefined`, you should guard against null reference exception:
```TypeScript
let errText = codesignResult.errors || "";
```